src/electron/electron/search.cljs

```diff
@@ -273,17 +273,33 @@
         (->>
          (concat matched-result
                  (search-blocks-aux database non-match-sql non-match-input page limit))
-         (distinct-by :id)
+         (distinct-by :rowid)
          (take limit)
          (vec))))))
 
+(defn- snippet-by
+  [content length]
+  (str (subs content 0 length) (when (> (count content) 250) "...")))
```

Comment:
should 250 replace by length arg?

Reply:
Here is to cut the results that fts5 snippet is not available. The snippet provided by fts5 only has 64 chars at max
So the default length of `10000` for block search doesn't fit this. 🤣